buildscripts/build.cmd

```diff
@@ -19,6 +19,13 @@ IF NOT EXIST %~dp0..\Settings.proj GOTO msbuild_not_configured
 REM Set Framework version based on passed in parameter
 IF "%1" == "" goto no_nothing
 
+REM nesting the netcore build in an IF body prevents reporting of the error level

```

Comment:
… and if I told you how long it took me to figure _that_ out, you wouldn't believe me.